src/interfaces/SpokePool.ts

```diff
@@ -105,7 +105,7 @@ export interface V3Fill extends V3RelayData {
   destinationChainId: number;
   relayer: string;
   repaymentChainId: number;
-  updatableRelayData: V3RelayExecutionEventInfo;
+  relayExecutionInfo: V3RelayExecutionEventInfo;
```

Comment:
@pxrl This was a bug I found once I sent a real fillV3 to the SpokePool (not via the MockSpkePoolClient) since the onchain event has the `relayExecutionInfo` prop not `updatableRelayData`.

Reply:
@nicholaspai I'd pulled that out from this PR and have proposed separately: https://github.com/across-protocol/sdk-v2/pull/550